includes/application/application.php

```diff
@@ -47,10 +47,10 @@
 }
 
 if( $flagSD )
-	echo "<tr><th>Start Date :</th><td> <input type=\"date\" name=Start_Date placeholder=\"Date yyyy-mm-dd\" required/></td></tr>";
+	echo "<tr><th>Start Date :</th><td> <input type=\"date\" name=Start_Date id=Start_Date placeholder=\"Date yyyy-mm-dd\" required/></td></tr>";
 
 if( $flagED )
-	echo "<tr><th>End Date :</th><td> <input type=\"date\" name=End_Date placeholder=\"Date yyyy-mm-dd\" required/></td></tr>";
+	echo "<tr><th>End Date :</th><td> <input type=\"date\" name=End_Date id=End_Date placeholder=\"Date yyyy-mm-dd\" required/></td></tr>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line exceeds 120 characters; contains 139 characters

<!-- Comment by [SideCI](https://sideci.com) -->